apk-generator/v2/scripts/build_apk.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-./gradlew assembleRelease --info
+./gradlew assemble$3Release --info
```

Comment:
@shailesh351 debug build is generated when we run `assembleDebug`. I don't think it's generated for these two tasks. Plus... See the list of tasks I posted above. These are the only two Google play and f Droid specific assemble tasks.